Flow.Launcher.Core/Plugin/PluginsLoader.cs

```diff
@@ -58,55 +58,57 @@ private static IEnumerable<PluginPair> DotNetPlugins(List<PluginMetadata> source
+                    catch (Exception)
+                    {
+                        throw;
+                    }
 #else
-                        catch (Exception e) when (assembly == null)
-                        {
-                            PublicApi.Instance.LogException(ClassName, $"Couldn't load assembly for the plugin: {metadata.Name}", e);
-                        }
-                        catch (InvalidOperationException e)
-                        {
-                            PublicApi.Instance.LogException(ClassName, $"Can't find the required IPlugin interface for the plugin: <{metadata.Name}>", e);
-                        }
-                        catch (ReflectionTypeLoadException e)
-                        {
-                            PublicApi.Instance.LogException(ClassName, $"The GetTypes method was unable to load assembly types for the plugin: <{metadata.Name}>", e);
-                        }
-                        catch (Exception e)
-                        {
-                            PublicApi.Instance.LogException(ClassName, $"The following plugin has errored and can not be loaded: <{metadata.Name}>", e);
-                        }
+                    catch (Exception e) when (assembly == null)
+                    {
+                        PublicApi.Instance.LogException(ClassName, $"Couldn't load assembly for the plugin: {metadata.Name}", e);
+                    }
+                    catch (InvalidOperationException e)
+                    {
+                        PublicApi.Instance.LogException(ClassName, $"Can't find the required IPlugin interface for the plugin: <{metadata.Name}>", e);
+                    }
+                    catch (ReflectionTypeLoadException e)
+                    {
+                        PublicApi.Instance.LogException(ClassName, $"The GetTypes method was unable to load assembly types for the plugin: <{metadata.Name}>", e);
+                    }
+                    catch (Exception e)
+                    {
+                        PublicApi.Instance.LogException(ClassName, $"The following plugin has errored and can not be loaded: <{metadata.Name}>", e);
+                    }
 #endif
 
-                        if (plugin == null)
-                        {
-                            erroredPlugins.Add(metadata.Name);
-                            return;
-                        }
+                    if (plugin == null)
+                    {
+                        erroredPlugins.Add(metadata.Name);
+                        return;
+                    }
+
+                    plugins.Add(new PluginPair { Plugin = plugin, Metadata = metadata });
+                });
 
-                        plugins.Add(new PluginPair { Plugin = plugin, Metadata = metadata });
-                    });
                 metadata.InitTime += milliseconds;
+                PublicApi.Instance.LogInfo(ClassName, $"Constructor cost for <{metadata.Name}> is <{metadata.InitTime}ms>");
```

Comment:
Should this be debug? I don't think it's relevant to users unless they need to figure where the start up slowness is.